interfaces/web-animations-2.idl

```diff
@@ -9,6 +9,12 @@ partial interface AnimationTimeline {
     Animation play (optional AnimationEffect? effect = null);
 };
 
+[Exposed=Window]
+partial interface Animation {
+    attribute CSSNumberish?       startTime;
```

Comment:
I've sent https://github.com/web-platform-tests/wpt/pull/31017 so that this will be tested.